static/app/views/dashboards/widgetBuilder/buildSteps/thresholdsStep/thresholdsHoverWrapper.tsx

```diff
@@ -15,6 +15,9 @@ type Props = {
   type?: string;
 };
 
+const NEGATIVE_POLARITY_COLOR_ORDER = ['green400', 'yellow400', 'red400'] as const;
+const POSITIVE_POLARITY_COLOR_ORDER = ['red400', 'yellow400', 'green400'] as const;
```

Comment:
Not a big deal, but you could use `lodash` `.reverse()` to reverse the negative order to get the positive order